lib/active_shipping/carriers/usps.rb

```diff
@@ -642,6 +644,13 @@ def response_message(document)
       response_status_node(document).text
     end
 
+    def find_country_code_case_insensitive(name)
+      upcase_name = name.upcase
+      country = ActiveUtils::Country::COUNTRIES.detect { |c| c[:name].upcase == upcase_name }
+      raise ActiveShipping::Error, "No uppercase country found for #{name}" unless country
```

Comment:
I can remove that I guess. I wanted to distinguish it from the ActiveUtils error a bit but it is already an ActiveShipping error with a backtrace to this method.